### DISCUSSION TITLE
>=1.9.5 compatibility

### DISCUSSION THREAD

**Discussion by dairiki-ren** [jothepro/doxygen-awesome-css#102] [Q&A] [answered]
Hey @jothepro and of course, the community!

Woah, this theme is awesome! @jothepro  have already pointed out in a release that the theme is not really well-working with doxygen >=1.9.5. I am a newbie with doxygen, and I would really love to use this theme with my project.

Is there a plan for adding compatibilty with newer doxygen versions? Or should I use an older version of doxygen? Are there any downsides?

All the best, 
zhengliw

**Comment by jothepro** [comment, marked as answer]
Thank you for your awesome feedback! 😊 

1.9.5 is not supported because of an issue in Doxygen that will be resolved in the upcoming release. 
If you want to make it work anyways, I've described a possible workaround in https://github.com/jothepro/doxygen-awesome-css/issues/86

I am planning to add support for the next doxygen release, once it is published. At the moment I recommend sticking to v1.9.4 if possible.

**Reply by dairiki-ren** [reply]
Okay, then I have no further concerns and will take a look at v1.9.4. Thanks for the answer, have a great day!
